src/composables/useNumbers.ts

```diff
@@ -5,6 +5,7 @@ import { useStore } from 'vuex';
 export type Preset =
   | 'default'
   | 'token'
+  | 'token_4_decimals'
```

Comment:
wasn't sure if this is the best name for this format... essentially a "fixed" 4 decimal format.

Reply:
```suggestion
  | 'token_fixed'
```